docs/source/cloud-setup/cloud-permissions/aws.rst

```diff
@@ -4,36 +4,330 @@
+
+  You may have previously set ``AWS_PROFILE`` in your ``.bashrc`` file or similar. Try to double-check the value:
+
+  .. code-block:: console
+      :emphasize-lines: 5
+
+      $ # See the currently enabled profile.
+      $ aws configure list
+            Name                    Value             Type    Location
+            ----                    -----             ----    --------
+         profile     AWSPowerUserAccess-xxxxxxx        env    ['AWS_DEFAULT_PROFILE', 'AWS_PROFILE']
+      access_key     ****************xxxx shared-credentials-file
+      secret_key     ****************xxxx shared-credentials-file
+          region                <not set>             None    None
+      $ # See the currently enabled AWS account and user/role.
+      $ aws sts get-caller-identity
+
+      $ # See if the environment variable has been set.
+      $ echo $AWS_PROFILE
+      AWSPowerUserAccess-xxxxxxx
+      $ # This profile will be used.
+      $ sky check aws
+
+      $ unset AWS_PROFILE
+      $ # Now, default profile will be used.
+      $ aws configure list
+            Name                    Value             Type    Location
+            ----                    -----             ----    --------
+         profile                <not set>             None    None
+         ...
+      $ sky check aws
+      $ # Delete from .bashrc/.zshrc to make the change permanent.
+
+- **Profile is not set**. If ``sky check aws`` and ``aws configure list`` cannot find credentials, you may not have a default profile set.
+
+  1. If the environment variable ``AWS_PROFILE`` is set, this profile name will be used.
+  2. If there is a profile named ``default``, it will be used.
+  3. Otherwise, the profile will not be accessible.
+
+  Note that even if there is only one profile, it will not be used unless ``AWS_PROFILE`` is set or the profile is named ``default``.
+
+  In AWS CLI v1, you can check ``~/.aws/credentials`` and ``~/.aws/config`` to look for profile names. In AWS CLI v2, you can check from the CLI.
+
+  .. code-block:: console
+
+      $ # AWS CLI v2 only
+      $ aws --version
+      aws-cli/2.25.6 ...
+
+      $ # List all profiles
+      $ aws configure list-profiles
+      AWSPowerUserAccess-xxxxxxx
+      default
+
+  If there is no ``default`` profile, you can edit the configuration directly:
+
+  .. code-block:: cfg
+      :emphasize-lines: 2
+
+      # ~/.aws/config
+      [profile default]
+      sso_session = my-skypilot-session
+      sso_account_id = XXXXXXXXXX
+      ...
+
+  .. code-block:: cfg
+      :emphasize-lines: 2
+
+      # ~/.aws/config
+      [default]
+      aws_access_key_id = XXXXXXXXXXXXXXXXXXXX
+      aws_secret_access_key = XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+
+  Or, you can set the ``AWS_PROFILE`` environment variable in your shell config:
+
+  .. code-block:: shell
+
+      # .bashrc / .zshrc
+      export AWS_PROFILE='AWSPowerUserAccess-xxxxxxx'
+
```

Comment:
We may want to add an API server item here for switching account issue with a remote API server. As discussed offline, we can fix the local API server for taking in `env_vars` starts with `AWS_`